### PR TITLE
drf start

### PR DESCRIPTION
**Pull request by Josephtobi**
Api post url built with basic authentication required 

Whats left?
Authentication for the api

### CODE REVIEW

File: app/repeater/settings.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(int(os.environ.get('DEBUG', 0)))
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```
Comment (tarxvftech) [inline app/repeater/settings.py:38]: Shouldn't need this wildcard for development (as long as DEBUG is true, i think), and it shouldn't be included in production as far as I know.
Is there cause to include this, or should it be removed?

File: app/repeater/settings.py [hunk 2]
```diff
@@ -131,3 +133,6 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+APPEND_SLASH=False
```
Comment (tarxvftech) [inline app/repeater/settings.py:138]: If I understand correctly, this means there won't be automatic redirects from something like `/rest/repeater` to `/rest/repeater/`.
Why do we want to include it? (To be clear, I'm not arguing against it, just making sure I understand! :D )

**Review by tarxvftech — CHANGES_REQUESTED** [review]
Please add .DS_Store to the top-level .gitignore and `git rm --cached` the .DS_Store files so they aren't tracked by git anymore. My fault for not using a default .gitignore that would catch that. If you see this on other projects, you might like https://stackoverflow.com/q/35581983

Thanks for your good work, I'm looking forward to merging it! :D

File: app/repeater/settings.py (hunk 2 above)
Comment (Josephtobi) [reply app/repeater/settings.py:138]: i had an issue with post request turning get
later resolved it with duplicate urls.
would remove this in my next pr attempt

File: app/repeater/settings.py (hunk 1 above)
Comment (Josephtobi) [reply app/repeater/settings.py:38]: it should be removed
but just to confirm, allow host is a list of domains that can call the api right?

**Pull request closed without merging**